src/exla/BUILD

```diff
@@ -19,7 +19,15 @@ cc_library(
     "@org_tensorflow//tensorflow/stream_executor:stream_executor_headers",
     "@org_tensorflow//tensorflow/compiler/xla:statusor",
   ],
-  linkstatic = 1,
+)
+
+# I don't like the NIF dependency
```

Comment:
Is this a TODO? :) Can we do something about it?

Reply:
Let me mark it as a TODO...